src/components/FilterTop.js

```diff
@@ -144,196 +144,199 @@ function Filter(props) {
   return (
     <React.Fragment>
       {
-        <Grid container wrap="nowrap" direction="row">
-          <Grid item className={classes.inputContainer}>
-            <TextField
-              select
-              label="Approved"
-              value={
-                approved === undefined
-                  ? 'All'
-                  : approved === true
-                  ? 'true'
-                  : 'false'
-              }
-              onChange={(e) =>
-                setApproved(
-                  e.target.value === 'All'
-                    ? undefined
-                    : e.target.value === 'true'
-                    ? true
-                    : false,
-                )
-              }
-            >
-              {['All', 'true', 'false'].map((name) => (
-                <MenuItem key={name} value={name}>
-                  {name}
-                </MenuItem>
-              ))}
-            </TextField>
-            <TextField
-              select
-              label="Rejected"
-              value={
-                active === undefined
-                  ? 'All'
-                  : active === true
-                  ? 'false'
-                  : 'true'
-              }
-              InputLabelProps={{
-                shrink: true,
-              }}
-              onChange={(e) =>
-                setActive(
-                  e.target.value === 'All'
-                    ? undefined
-                    : e.target.value === 'true'
-                    ? false
-                    : true,
-                )
-              }
-            >
-              {['All', 'false', 'true'].map((name) => (
-                <MenuItem key={name} value={name}>
-                  {name}
-                </MenuItem>
-              ))}
-            </TextField>
-            <MuiPickersUtilsProvider
-              utils={DateFnsUtils}
-              locale={getDatePickerLocale()}
-            >
-              <KeyboardDatePicker
-                margin="normal"
-                id="start-date-picker"
-                label="Start Date"
-                format={getDateFormatLocale(true)}
-                value={dateStart}
-                onChange={handleDateStartChange}
-                maxDate={dateEnd}
-                KeyboardButtonProps={{
-                  'aria-label': 'change date',
+        <form onSubmit={handleSubmit}>
```

Comment:
The `form` approach is causing the page to reload for me.

https://user-images.githubusercontent.com/5558838/117713347-78523e80-b1cd-11eb-8e85-edc1c2a511a9.mov